docs/website/docs/command-reference/link.md

```diff
@@ -0,0 +1,383 @@
+---
+title: odo link
+sidebar_position: 4
+---
+
+`odo link` command helps link an odo component to an Operator backed service or another odo component. It does this by using [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator). At the time of writing this, odo makes use of the Service Binding library and not the Operator itself to achieve the desired functionality.
```

Comment:
This documentation is very detailed, and based on the quickstart project: I think it would be better to place it in the **Using odo** section, and have a more concise and generic doc as reference

Reply:
Yes, while writing this, I had the thought of putting a concise form of it in the "Using odo" section. But can we do it separately?

Also, I think only this doc in "Command Reference" is based on Quickstart project. Which is not good. But I couldn't think of a better way to convey the point. Do you have any other idea?

Reply:
Ah, I see your point now. You are suggesting to add this document to the "Using odo" section. And in this section, we keep a concise document.

We will have to add the bits for linking backend and frontend component in that case. And also figure out what to keep in what doc. Doable. Let's do it.